aws/resource_aws_security_group_rule_test.go

```diff
@@ -1922,7 +1921,7 @@ resource "aws_security_group_rule" "test" {
 `, rName, description)
 }
 
-func testAccAWSSecurityGroupRuleConfigDescriptionAllPortsToPort65535(rName, description string) string {
+func testAccAWSSecurityGroupRuleConfigDescriptionAllPortsNonZeroPorts(rName, description string) string {
```

Comment:
Non-zero for both `from_port` and `to_port` also covers the previous scenario, which is why this was updated.